src/config.ts

```diff
@@ -12,8 +12,6 @@ import fs from 'fs';
 import { from } from 'env-var';
 import yaml from 'js-yaml';
 
-require('dotenv').config();
```

Comment:
Does it work if we remove dotenv here?

Reply:
We don't need to use `dotenv` in production env, we use k8s ConfigMaps/Secrets for setting all needed env vars. So I should have this line in code.  To use .env files locally we can utilize a better approach:
https://github.com/pm4ml/mojaloop-payment-manager-management-api/pull/32/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R23

And yes - it works on k8s env:
![image](https://github.com/user-attachments/assets/37b507a8-623d-444f-b620-41fe02ef8bb7)

Reply:
Oh ok. If it works without importing, then that's ok. Infact that is my question, got it.